src/stores/feature-flag.ts

```diff
@@ -106,10 +106,12 @@ export const useFeatureFlagStore = defineStore(FEATURE_FLAG, {
      * @param cookies - mapping of feature flags and their preferred states
      */
     initFromCookies(cookies: Record<string, FeatureState>) {
-      Object.entries(this.flags).forEach(([name, flag]) => {
-        if (getFlagStatus(flag) === SWITCHABLE)
+      for (const flagEntry of Object.entries(this.flags)) {
+        const [name, flag] = flagEntry as [string, FeatureFlag]
+        if (getFlagStatus(flag) === SWITCHABLE) {
           flag.preferredState = cookies[name]
-      })
+        }
+      }
```

Comment:
Is this a change to appease TypeScript?

I don't remember if this is still the case, but at one point using a `for...of` loop actually incurred a non-trivial size penalty due to the generator runtime transpilation of it. Might not be an issue any more though.

Reply:
Yes, it's TS appeasing change. I didn't know about for ... of performance. I think I can improve this code by typing `this`